### PR TITLE
Home.js

### PR DESCRIPTION
**Pull request by joseteranc**
Working on App.js
Start Date
Return Date

### CODE REVIEW

File: src/components/ReturnDate.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+`;
+
+export default function StartDateField() {
+  return (
+    <ReturnDate for="start">
+      Retrun Date:
+      <input
+        type="date"
+        id="start"
+        name="trip-start"
```
Comment (marwinburesch) [inline src/components/ReturnDate.js]: this should probably be named "trip-end"

File: src/stories/ReturnDateFiel.stories.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import ReturnDate from "../components/ReturnDate";
+
+export default {
+  title: "ReturnDate"
+};
+
+export const ReturnDateField = () => <ReturnDate></ReturnDate>;
```
Comment (marwinburesch) [inline src/stories/ReturnDateFiel.stories.js]: The filename is missing a "d", and you could have these two components consolidated in one component, like `SelectTripPeriod` or something and then just have one story for that. `StartDateField` and `ReturnDateField` will probably always used together, so might as well put them in one component

**Comment by marwinburesch** [comment]
Try to include a screenshot with your pull requests, that way it's easier to read the code as well.

otherwise it's looking good mate

File: src/components/ReturnDate.js (hunk 1 above)
Comment (joseteranc) [reply src/components/ReturnDate.js]: check thanks

File: src/stories/ReturnDateFiel.stories.js (hunk 2 above)
Comment (joseteranc) [reply src/stories/ReturnDateFiel.stories.js]: Thank you Marwin,

**Pull request merged**